### PR TITLE
Check for errors when trying to create the auth token file

### PR DESCRIPTION
**Pull request by danielfm**
@mumoshu addressing your comment in my last PR #439.

### CODE REVIEW

File: core/controlplane/config/token_config.go [hunk 1]
```diff
@@ -156,10 +156,11 @@ func ReadOrEncryptAuthTokens(dirname string, encryptor CachedEncryptor) (*Encryp
 
 	// Auto-creates the auth token file, useful for those coming from previous versions of kube-aws
 	if _, err := os.Stat(authTokenPath); os.IsNotExist(err) {
-		_, err := os.OpenFile(authTokenPath, os.O_RDONLY|os.O_CREATE, 0600)
+		file, err := os.OpenFile(authTokenPath, os.O_RDONLY|os.O_CREATE, 0600)
 		if err != nil {
 			return nil, err
 		}
+		defer file.Close()
```
Comment (redbaron) [inline core/controlplane/config/token_config.go]: you never use `file`, why `defer`?

Comment (danielfm) [inline core/controlplane/config/token_config.go]: @redbaron I just wanted to make sure all handles were closed properly. Is this unnecessary in this case?

Comment (redbaron) [inline core/controlplane/config/token_config.go]: `defer` delays `file.Close()` until function exits. you don't use `file` anywhere, so close it right there, without `defer`

Comment (danielfm) [inline core/controlplane/config/token_config.go]: @redbaron done! Thanks for your feedback.

**Comment by mumoshu** [comment]
LGTM 👍 
Thanks for contributing @danielfm, and for reviewing @redbaron :bow:

**Pull request merged**